packages/server/assets/multiregion/typedefs/multiregion.graphql

```diff
@@ -2,7 +2,7 @@ type ServerRegionItem {
   id: String!
   key: String!
   name: String!
-  description: String!
+  description: String
```

Comment:
Do we throw if creating a workspace invite using old mutation?

Reply:
1. we should, but i have not yet done that
2. its not yet updated, that is why i asked for help in the morning,